agent/functional_tests/tests/functionaltests_windows_test.go

```diff
@@ -0,0 +1,368 @@
+// +build windows
+// Copyright 2014-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package functional_tests
+
+import (
+	//"bytes"
+	"fmt"
+	//"io/ioutil"
+	//"os"
+	//"path/filepath"
+	"reflect"
+	//"regexp"
+	//"strconv"
+	"strings"
+	"testing"
+	"time"
+	"os"
+
+	//"github.com/aws/amazon-ecs-agent/agent/ecs_client/model/ecs"
+	. "github.com/aws/amazon-ecs-agent/agent/functional_tests/util"
+	//"github.com/aws/amazon-ecs-agent/agent/utils"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	//"github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	//docker "github.com/fsouza/go-dockerclient"
+	//"github.com/pborman/uuid"
+)
+
+const (
+	waitTaskStateChangeDuration     = 2 * time.Minute
+	waitMetricsInCloudwatchDuration = 4 * time.Minute
+	awslogsLogGroupName             = "ecs-functional-tests"
+)
+
+// TestRunManyTasks runs several tasks in short succession and expects them to
+// all run.
+func TestRunManyTasks(t *testing.T) {
+fmt.Println("starting")
```

Comment:
there are a number of these in this file. You might wanto to get rid of this. Or, use log instead